esphome/components/xiaomi_ble/xiaomi_ble.h

```diff
@@ -41,6 +42,7 @@ struct XiaomiParseResult {
   optional<bool> is_active;
   optional<bool> has_motion;
   optional<bool> is_light;
+  optional<bool> is_lock;
```

Comment:
`open` is probably better than `lock` as this does not tell you if something is locked, but open or closed.
```suggestion
  optional<bool> is_open;
```